<!DOCTYPE html>
<html>
<head>
    <meta charset="utf-8">
    <title>HPC-T-Annotator Interface Guide</title>
    <link rel="stylesheet" type="text/css" href="static/styles.css">
    <link rel="icon" type="image/x-icon" href="static/logo.ico">
    <script src="static/scripts.js" defer></script>
</head>
<body>

    <nav>
        <ul>
            <li><a href="index.html">Home</a></li>
            <li><a href="start.html">Start</a></li>
            <li><a href="tool.html">CLtool</a></li>
            <li><a href="architecture.html">Architecture</a></li>
            <li><a href="notebooks.html">Notebooks</a></li>
            <li>
                <!-- <a href="help.html">Help</a> --><a>Help</a>
                <ul class="dropdown-content">
                    <li><a href="interface_guide.html">Interface guide</a></li>
                    <li><a href="example.html">Step-by-step example</a></li>
                </ul>
            </li>
            <li><a href="contacts.html">Contacts</a></li>
        </ul>
    </nav>

<div class="container">
    <div class="title">
        <div class="best" style="display: flex; align-items: center;">
            <img src="static/logo.png" width="100" style="margin: auto;">
            <h1>HPC-T-Annotator</h1>
            <img src="static/logo.png" width="100" style="margin: auto;">
        </div>
        <h2 id="subtitle">Post-processing Notebooks</h2>
    </div>

    <div class="description">
        <p>Interpreting extensive biological datasets, generated from a variety of analysis and alignment tools, is complicated by the complexity of data visualization. Traditional methods often fall short in capturing the nuances of these intricate biological systems, highlighting the need for innovative graphical representation approaches.

            Given that transcriptome annotation produces large alignment data files, accurately interpreting this information is essential. Therefore, these notebooks tackle the crucial task of extracting and visualizing relevant information from this data, aiding in its accurate interpretation.</p>
    </div>

    <div class="block">
        <h3 style="padding: -100px;">AnnoDegsReport: Multi-database Annotation Summary</h3>
        <div><h4>Group: Transcriptomic data analysis</h4></div>
        <p><strong>Input:</strong> ALL BLAST/Diamond alignment TSV results (blastx and blastp vs all databases) and Deseq2 output table.</p>
        <p>This notebook is capable of generating summary reports in XSLX or TSV format, consolidating the annotation results of a specific transcriptome obtained by running alignment software (Diamond and BLAST) against various sequence databases. Additionally, this notebook can create links to the accession numbers for the input transcript sequences that mapped to the database, as well as resources on the NCBI and UniProt portals.</p>
        <a class="github-button" href="https://github.com/lorenzo-arcioni/BioSwiftAnalytics/blob/main/notebooks/AnnoDegsReport.ipynb" target="_blank">Open on GitHub</a>
    </div>

    <div class="block">
        <h3 style="padding: -100px;">AnnoRate: Hit Rate Comparison</h3>
        <div><h4>Group: Transcriptomic data analysis</h4></div>
        <p><strong>Input:</strong> Input transcriptome (pep and cds) and Diamond/BLAST alignment results (in TSV format).</p>
        <p>This notebook analyzes the hit percentages for sequences across various databases. It evaluates the performance of a database in comparison to an input transcriptome. The notebook accepts a Fasta-format transcriptome file and a TSV-format alignment output from tools like Diamond or BLAST. It calculates hit percentages and generates a summary table, aiding users in assessing database coverage and specificity. This analysis helps select the most suitable database for a specific analysis.</p>
        <a class="github-button" href="https://github.com/lorenzo-arcioni/BioSwiftAnalytics/blob/main/notebooks/AnnoRate.ipynb" target="_blank">Open on GitHub</a>
    </div>

    <div class="block">
        <h3 style="padding: -100px;">AnnoViz: A Jupyter Notebook for Enhanced Annotation Result Interpretation</h3>
        <div><h4>Group: Transcriptomic data analysis</h4></div>
        <p><strong>Input:</strong> One single BLAST/Diamond alignment TSV result.</p>
        <p>This notebook takes as input the output file generated by annotation software (BLAST or Diamond) and creates graphs that provide an interpretation of the annotation and alignment results. It offers valuable insights into the transcripts that have mapped to the database.</p>
        <a class="github-button" href="https://github.com/lorenzo-arcioni/BioSwiftAnalytics/blob/main/notebooks/AnnoViz.ipynb" target="_blank">Open on GitHub</a>
    </div>

    <div class="block">
        <h3 style="padding: -100px;">MultiVenn: A Jupyter Notebook for Comparing Annotation Results Across Databases Using Venn Diagrams</h3>
        <div><h4>Group: Transcriptomic data analysis</h4></div>
        <p><strong>Input:</strong> Species specific Diamond/BLAST annotation TSV results across multiple databases.</p>
        <p>Venn diagrams are a powerful visualization tool used in many areas of research, including bioinformatics. They are typically used to show the overlap between different sets or categories of data. In the context of omics data analysis, Venn diagrams can help identify common or unique hits across multiple databases or analyses, providing insights into the relationships between different datasets. In addition to the traditional Venn diagram, there are also variations such as Euler diagrams and Edwards-Venn diagrams, which can provide different types of visual representations of the data.</p>
        <a class="github-button" href="https://github.com/lorenzo-arcioni/BioSwiftAnalytics/blob/main/notebooks/MultiVenn.ipynb" target="_blank">Open on GitHub</a>
    </div>
    
    

</div>
</body>
</html>